src/components/Badge/Badge.tsx

```diff
@@ -51,13 +51,19 @@ export const Badge: FunctionComponent<BadgeProps> = ({
         `moonstone-${type}`,
         className
     );
+
+    if (!label || label.length < 1) {
```

Comment:
Can't the diamond type of the Badge be used without a label?

Reply:
You right!
I'm wondering if this variant is used, I'd like to remove it.

Reply:
I haven't seen it being used: https://github.com/search?q=org%3AJahia+type%3D%22diamond%22&type=code